ssl/record/rec_layer_s3.c

```diff
@@ -870,10 +870,7 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
             rectype = type;
         SSL3_RECORD_set_type(thiswr, rectype);
 
-        /*
-         * Some servers hang if initial client hello is larger than 256 bytes
-         * and record version number > TLS 1.0
-         */
+        /* Some servers hang if record version number > TLS 1.0 */
```

Comment:
I would not change this comment. The information is still relevant although we explicitly do not check the > 256 bytes length.

Reply:
`initial client hello is larger than 256 bytes and record version number > TLS 1.0` are these two conditions parallel? 
They also paid attention to these two issues  https://github.com/openssl/openssl/issues/9073  https://github.com/openssl/openssl/issues/1689

Reply:
Sorry, I misunderstood the possible conditions. and Will remove commit.